src/App.css

```diff
@@ -30,3 +31,114 @@
     transform: rotate(360deg);
   }
 }
+
+/** Scafolding **/
+
+.page {
```

Comment:
Why shouldn't this be an id selector instead, since it is only going to be used by only one element? Same comment applies to other classes that are only used by one element ex. rightpanel, header, footer.

Reply:
It can be. Converted almost all classes to ids because pretty much all of them are single use styles.